backend/api/tests/test_models.py

```
@@ -0,0 +1,222 @@
+import pytest
+from django.core.exceptions import ValidationError
+from api.models import User, CustomUserManager, CompanyProfile, ViewerProfile, Video
+
+@pytest.mark.django_db
+class TestUserModel:
+    def test_create_user(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_create_user`

Add return type annotation: `None`

(ANN201)